.github/workflows/docker.yml

```diff
@@ -85,6 +91,13 @@ jobs:
           build-args: |
             SERVER_IMAGE=temporaliotest/server:${{ steps.build_args.outputs.image_tag }}
 
+      - name: Run Trivy vulnerability scanner on Admin Tools image
+        if: steps.build_args.outputs.push == 'true'
+        uses: ./.github/actions/trivy
+        with:
+          image-tags: ${{ steps.meta_admin_tools.outputs.tags }}
+          image-name: admin_tools
```

Comment:
isn't it `admin-tools`?

Reply:
This name doesn't have any meaning except logging. I don't have any preferences here. I can change the formatting.

Reply:
if it doesn't have any meaning, how does the action know what image to scan? is `tags` a full name including repository and tag?

Reply:
Yes, `tags` contain full image names, for example:
```
temporaliotest/server:latest
temporaliotest/server:sha-24fcf99
```